test/e2e/create_test.go

```diff
@@ -40,6 +42,78 @@ func creationTestSuite(t *testing.T) {
 	t.Run("Network validation", testNetwork)
 	t.Run("Label validation", func(t *testing.T) { testWorkerLabel(t) })
 	t.Run("NodeTaint validation", func(t *testing.T) { testNodeTaint(t) })
+	t.Run("machineCreation", func(t *testing.T) { testWindowsMachineCreation(t) })
```

Comment:
We should stick with the general naming scheme for tests here, using spaces.

Reply:
I'm pretty sure this test will be changed pretty soon, it looks pretty out of place right now, but that wont be the case soon right?